samples/charts/bar/float.html

```diff
@@ -0,0 +1,158 @@
+					[randomScalingFactor(), randomScalingFactor()],
+					[randomScalingFactor(), randomScalingFactor()],
+					[randomScalingFactor(), randomScalingFactor()],
+					[randomScalingFactor(), randomScalingFactor()]
+				]
+			}, {
+				label: 'Dataset 2',
+				backgroundColor: color(window.chartColors.blue).alpha(0.5).rgbString(),
+				borderColor: window.chartColors.blue,
+				borderWidth: 1,
+				data: [
+					[randomScalingFactor(), randomScalingFactor()],
+					[randomScalingFactor(), randomScalingFactor()],
+					[randomScalingFactor(), randomScalingFactor()],
+					[randomScalingFactor(), randomScalingFactor()],
+					[randomScalingFactor(), randomScalingFactor()],
+					[randomScalingFactor(), randomScalingFactor()],
+					[randomScalingFactor(), randomScalingFactor()]
+				]
+			}]
+
+		};
+
+		window.onload = function() {
+			var ctx = document.getElementById('canvas').getContext('2d');
+			window.myBar = new Chart(ctx, {
+				type: 'bar',
+				data: barChartData,
+				options: {
+					responsive: true,
+					legend: {
+						position: 'top',
+					},
+					title: {
+						display: true,
+						text: 'Chart.js Bar Chart'
+					},
+					tooltips: {
+						callbacks: {
+							label: function(tooltipItem) {
+								var label = tooltipItem.label || '';
+								var custom = tooltipItem.custom || {};
+
+								if (label) {
+									label += ': ';
+								}
+								label += ' [' + custom.start + ', ' + custom.end + ']';
+								return label;
+							}
+						}
+					}
+				}
+			});
+
+		};
+
+		document.getElementById('randomizeData').addEventListener('click', function() {
+			var zero = Math.random() < 0.2 ? true : false;
+			barChartData.datasets.forEach(function(dataset) {
+				dataset.data = dataset.data.map(function() {
+					return zero ? 0.0 : [randomScalingFactor(), randomScalingFactor()];
+				});
+
+			});
+			window.myBar.update();
+		});
+
+		var colorNames = Object.keys(window.chartColors);
+		document.getElementById('addDataset').addEventListener('click', function() {
+			var colorName = colorNames[barChartData.datasets.length % colorNames.length];
+			var dsColor = window.chartColors[colorName];
+			var newDataset = {
+				label: 'Dataset ' + (barChartData.datasets.length + 1),
+				backgroundColor: color(dsColor).alpha(0.5).rgbString(),
+				borderColor: dsColor,
+				borderWidth: 1,
+				data: []
+			};
+
+			for (var index = 0; index < barChartData.labels.length; ++index) {
+				newDataset.data.push([randomScalingFactor(), randomScalingFactor()]);
+			}
+
+			barChartData.datasets.push(newDataset);
+			window.myBar.update();
+		});
+
+		document.getElementById('addData').addEventListener('click', function() {
+			if (barChartData.datasets.length > 0) {
+				var month = MONTHS[barChartData.labels.length % MONTHS.length];
+				barChartData.labels.push(month);
+
+				for (var index = 0; index < barChartData.datasets.length; ++index) {
+					// window.myBar.addData(randomScalingFactor(), index);
```

Comment:
commented code

Reply:
Removed (also from vertical.html, that I used as template)